pkg/shell/shell-modals.jsx

```diff
@@ -86,6 +86,8 @@ export const AboutCockpitModal = () => {
 
 export const LangModal = () => {
     const language = document.cookie.replace(/(?:(?:^|.*;\s*)CockpitLang\s*=\s*([^;]*).*$)|^.*$/, "$1") || "en-us";
+    const lang_dir = language.match(/he-il/) ? "rtl" : "ltr";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-14107-20221017-161445-5d065362-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)